tests/test_filling_fields.py

```diff
@@ -36,405 +34,385 @@
 )
 
 
-def assert_not_raise(method, parameters, exception):
-    try:
-        method(*parameters)
-    except exception:
-        msg = "Exception %s not expected to be raised" % exception.__name__
-        raise AssertionError(msg)
-
-
 @pytest.fixture
 def person(db):
     return mommy.make('generic.Person')
 
 
-class FieldFillingTestCase(TestCase):
-
-    def setUp(self):
-        self.person = mommy.make(models.Person)
+@pytest.fixture()
+def custom_cfg():
+    yield None
+    if hasattr(settings, 'MOMMY_CUSTOM_FIELDS_GEN'):
+        delattr(settings, 'MOMMY_CUSTOM_FIELDS_GEN')
+    mommy.generators.add('tests.generic.fields.CustomFieldWithGenerator', None)
+    mommy.generators.add('django.db.models.fields.CharField', None)
 
 
-class FillingFromChoice(FieldFillingTestCase):
+class TestFillingFromChoice():
 
-    def test_if_gender_is_populated_from_choices(self):
+    def test_if_gender_is_populated_from_choices(self, person):
         from tests.generic.models import GENDER_CH
```

Comment:
Would be nice to rename this variable at some point. This `CH` is not clear for me.